.github/workflows/release.yml

```diff
@@ -45,7 +45,7 @@ jobs:
         run: node ./scripts/get-version.js
       - name: Commit changes, tag and push
         run: |
-          git commit --all -m "[Version bump] chore(release): publish v${{ env.tag }}"
+          git commit --all -m "[Version bump] chore(release): publish v${{ env.tag }}\nlerna.json\n\nCHANGELOG.md\n**/CHANGELOG.md"
```

Comment:
why 2 `\n`?

Reply:
Good question!
This is because the Package roll out certifier want's it that way simply
https://coveord.atlassian.net/wiki/spaces/CM/pages/1786708612/Package+Rollout+Certifier